packages/flutter_tools/test/general.shard/forbidden_imports_test.dart

```diff
@@ -2,22 +2,24 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-import 'package:flutter_tools/src/base/file_system.dart';
-import 'package:flutter_tools/src/globals.dart' as globals;
+import 'package:file/file.dart';
+import 'package:file/local.dart';
 
 import '../src/common.dart';
 
+const FileSystem fileSystem = LocalFileSystem();
```

Comment:
Doesn't have to be in this PR, but should this be moved to the integration shard?

Reply:
I wasn't sure. It is definitely touching the real filesystem, but it is also more of an analysis check - I could see wanting it to remain fast.

Reply:
Does it run in parallel with other general shard tests?

Reply:
yes ... hopefully those aren't also writing to the filesytem...hmm

Reply:
we should probably just move this?

Reply:
i vote move them just to be safe and rule out the potential flaking.

Reply:
done!